.github/workflows/publish.yml

```diff
@@ -0,0 +1,28 @@
+name: Publish
+
+on:
+  push:
+    tags:
+      - '*.*.*'
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    # Setup .npmrc file to publish to npm
+    - uses: actions/setup-node@v1
+      with:
+        node-version: '12.x'
+        registry-url: 'https://registry.npmjs.org'
+    - uses: rtCamp/action-slack-notify@master
+      env:
+        SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
+        SLACK_MESSAGE: Successfully published `@skyux-sdk/builder@${{ github.event.release.tag_name }}` to NPM.\nhttp://github.com/${{ github.repository }}/blob/${{ github.event.release.tag_name }}/CHANGELOG.md
+    - name: Install
+      run: |
+        npm install
+    - name: Publish
+      run: npm publish --dry-run
```

Comment:
Using `--dry-run` just to make sure it publishes correctly. We can remove this in a later PR.